DOCUMENTATION.md

```diff
@@ -181,6 +181,27 @@ email_reply_to_id='8e222534-7f05-4972-86e3-17c5d9f894e2' # optional UUID string
 
 If you omit this argument, the client will set your default email reply-to address for the notification.
 
+### Attach a document
+
+You can attach a document to an email notification.
+
+Contact the GOV.UK Notify team on the support page or through the Slack channel to enable this function for your service.
+
+The document must be a PDF file smaller than 2MB. To attach the document, you must:
+
+1. Define the PDF as a file object.
```

Comment:
I think this step is confusing and I'd argue it's not necessary. You don't define anything, you open an existing file, but that's not the only way to achieve this and it should be clear from the code sample.